.gitignore

```diff
@@ -8,7 +8,7 @@
 /.bundle
 
 # Ignore .dotenv config
-.env*
+.env
```

Comment:
leave the * there please; `.env.development` etc is valid for environment specific `.env` files and the wildcard will catch all of those

Reply:
Okay! I'm submitting a revision right now.